formats/frontend_links_definition.json

```diff
@@ -62,6 +62,9 @@
       "analytics_identifier": {
         "$ref": "#/definitions/analytics_identifier"
       },
+      "withdrawn": {
+        "type": "boolean"
```

Comment:
Can you add a description to this? Perhaps with a link to the publishing-api docs about what it means?